.github/workflows/nightly_release_testing.yaml

```diff
@@ -247,6 +247,7 @@ jobs:
       - ondemand-aws-tests
       - ondemand-gcp-tests
       - kubernetes-tests
+      - ondemand-aws-multinode-tests
```

Comment:
is this job defined somewhere? I can't seem to find it. and do we want to aws w/ https auth to nightly release testing as well?

Reply:
yea its defined lower down in the file, basically we're using it to run the tests inside `TestMultiNodeCluster`

the ondemand aws tests should be running with right now actually, using the `ondemand_aws_https_cluster_with_auth` fixture

Reply:
oops my local repo has not been updated in 2 weeks, looks good!